### PR TITLE
kubevpn: Add version 2.2.17

### PR DESCRIPTION
**Pull request by wencaiwulue**
Add package kubevpn: KubeVPN offers a Cloud Native Dev Environment that connects to kubernetes cluster network.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kubevpn.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "hash": "2050af3cd8e1e2a717a7f74610d881bad2bbae6d8ff4e7670c5c5ef2a95c8e9b",
+            "extract_dir": "bin"
+        },
+        "arm64": {
+            "url": "https://github.com/kubenetworks/kubevpn/releases/download/v2.2.17/kubevpn_v2.2.17_windows_arm64.zip",
+            "hash": "cce520fe2d44761d2ca679fbfaa886909d5664f8c3199aca8b06e75868b2bdc6",
+            "extract_dir": "bin"
+        }
+    },
+    "bin": "kubevpn.exe",
```
Comment (aliesbelik) [inline bucket/kubevpn.json:23]:
```suggestion
            "hash": "4513683410566628f39c2f423e066c4903bb7b3abd7a80a1ba6115e8cc65115d"
        },
        "32bit": {
            "url": "https://github.com/kubenetworks/kubevpn/releases/download/v2.2.17/kubevpn_v2.2.17_windows_386.zip",
            "hash": "2050af3cd8e1e2a717a7f74610d881bad2bbae6d8ff4e7670c5c5ef2a95c8e9b"
        },
        "arm64": {
            "url": "https://github.com/kubenetworks/kubevpn/releases/download/v2.2.17/kubevpn_v2.2.17_windows_arm64.zip",
            "hash": "cce520fe2d44761d2ca679fbfaa886909d5664f8c3199aca8b06e75868b2bdc6"
        }
    },
    "extract_dir": "bin",
    "bin": "kubevpn.exe",
```

File: bucket/kubevpn.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "version": "2.2.17",
+    "description": "KubeVPN offers a Cloud Native Dev Environment that connects to kubernetes cluster network.",
+    "homepage": "https://github.com/kubenetworks/kubevpn",
```
Comment (aliesbelik) [inline bucket/kubevpn.json:4]:
```suggestion
    "homepage": "https://www.kubevpn.cn/",
```

File: bucket/kubevpn.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_amd64.zip",
+                "extract_dir": "bin"
+            },
+            "32bit": {
+                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_386.zip",
+                "extract_dir": "bin"
+            },
+            "arm64": {
+                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_arm64.zip",
+                "extract_dir": "bin"
```
Comment (aliesbelik) [inline bucket/kubevpn.json:45]:
```suggestion
                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_amd64.zip"
            },
            "32bit": {
                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_386.zip"
            },
            "arm64": {
                "url": "https://github.com/kubenetworks/kubevpn/releases/download/v$version/kubevpn_v$version_windows_arm64.zip"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Comment by wencaiwulue** [comment]
@aliesbelik it's very kind of you, thianks a lot ~

**Comment by wencaiwulue** [comment]
fix 
```txt
Error: [-] files end with a newline 68ms (67ms|1ms)
```

by https://github.com/ScoopInstaller/Extras/commit/9c46f210d22b0144b77c92581225cff36f7e85ee

**Comment by wencaiwulue** [comment]
solve #13943

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**